bootcamp/models/user.py

```diff
@@ -0,0 +1,40 @@
+import uuid
+
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Text,
+)
+from sqlalchemy.dialects.postgresql import UUID
+
+from bootcamp.lib.validation import is_valid_uuid_string
+from bootcamp.models import logger
+from bootcamp.models.base import Model
+
+
+class User(Model):
+    __tablename__ = 'users'
+
+    id = Column(Integer, primary_key=True)
+    uuid = Column(UUID, default=lambda: str(uuid.uuid4()))
+
+    user_name = Column(String(20))
```

Comment:
Username and password should not be nullable, let's use 
```python
user_name = Column(String(20), nullable=False)
```
I'll do samething in `bootcamp/models/title.py`.